### DISCUSSION TITLE
Is there a way to query content items unpublished within the last 24 hours?

### DISCUSSION THREAD

**Discussion by bashuss** [OrchardCMS/OrchardCore#15340] [Q&A] [answered]
I can figure out a way to query ContentItemIndex that shows, if contentItems have been published in the last 24 hours. 
But I am having a hard to find a way to query if they have been unpublished within the last 24 hours.

My guess is, that a previously published content item is transformed into an item that is 
`published==false and publishedUtc!=null`

But the modifiedUtc is not enough to find out when it has been unpublished, as the item may have been unpublished a while ago but edited without publishing it.

Am I missing something?

**Comment by Skrypt** [comment, marked as answer]
I think you will need to use Audit Trail for that matter.

**Reply by bashuss** [reply]
Thank you, will try.
